### PR TITLE
AKD local auditor basic user-interface.

### PR DESCRIPTION
**Pull request by slawlor**
This is launched with

```bash
cargo run -p akd_local_auditor -- ui
```

which will let you set either Amazon S3 or dynamo DB settings for epoch storage.

**Main auditor page**
image

**Drilling down to a proof**
image

**Auditor settings**
image

**Logstream interface**
image

### CODE REVIEW

**Comment by slawlor** [comment]
@dillonrg feel free to push to this branch! We can collab here

**Comment by slawlor** [comment]
Making available for review as development can continue async!

File: akd_local_auditor/Cargo.toml [hunk 1]
```diff
@@ -5,6 +5,7 @@ version = "0.8.4"
 authors = ["Sean Lawlor <seanlawlor@fb.com>"]
 edition = "2018"
 publish = false
+# resolver = "2"
```
Comment (kevinlewi) [inline akd_local_auditor/Cargo.toml:8]: This can be removed right?

File: .vscode/launch.json [hunk 2]
```diff
@@ -4,7 +4,22 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
-
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "Debug AKD auditor User Interface",
```
Comment (kevinlewi) [inline .vscode/launch.json:10]: auditor -> Auditor

**Review by kevinlewi — APPROVED** [review]
Looks good to me!

**Pull request merged**